Time Tracker/Tracker.cs

```diff
@@ -11,30 +14,37 @@ public class Tracker
         public Tracker(List<Button>buttons)
         {
             this._buttons = buttons;
+            this._stopwatch = new Stopwatch();
+            this._log = new Log(_stopwatch);
+            this._csv = new CSV();
```

Comment:
???? Maybe instead of a CSV, you can have a interface of database and in case that you want a CSV you can pass this info in the constructor